manifold_airflow_dags/tupress_ingest_dag.py

```diff
@@ -84,6 +87,19 @@ def calculate_file_to_move(**context):
     dag=DAG
 )
 
+ingest_deltas_bash = """
```

Comment:
```suggestion
ingest_deltas_bash = f"""
```

I think this needs to be turned into a template in order to work?